neo/app/views/index.scala.html

```diff
@@ -7,16 +7,31 @@
 
   <time-series id="chart"></time-series>
 
-  <div id='sidebar' ng-init="click = -1" ng-class="{toggled: click === 1}" ng-click="click = click*-1">
+  <div id='sidebar' ng-init="click = -1; tab=3" ng-class="{toggled: click === 1}" ng-click="click = click*-1">
```

Comment:
this blog may help: https://blog.twitter.com/2014/rendering-tweets-with-angularjs-and-nodejs